lib/zoninator_rest/type/class-zoninator-rest-type-registry.php

```diff
@@ -40,7 +40,7 @@ class Zoninator_REST_Type_Registry {
 	 *
 	 * @throws Zoninator_REST_Exception When $instance not a Zoninator_REST_Interfaces_Type.
 	 */
-	public function define( $identifier, $instance ) {
+	public function define( $identifier, $instance ) { // phpcs:ignore WordPressVIPMinimum.Constants.ConstantString.NotCheckingConstantName
```

Comment:
This seems likes a false positive cc @rebeccahum

Reply:
I scanned this file locally and I'm not getting that error. Are you, @GaryJones?

Reply:
I didn't scan it, but went on the assumption that if @vaurdan was adding the ignore, that he might have been.

Reply:
I just double-checked the version being used and it was 2.2. Updating it to 2.3.2 seems to have removed the Error, so it is a false positive. 

With this, I have noticed that we don't have a VIPCS dependency on composer.json, so I'm going to add it to prevent issues like this.